worldedit-core/src/main/java/com/sk89q/worldedit/regions/selector/EllipsoidRegionSelector.java

```diff
@@ -208,6 +209,7 @@ public EllipsoidRegion getIncompleteRegion() {
 
     @Override
     public void learnChanges() {
+        this.started = !region.getCenter().equals(Vector3.ZERO);
```

Comment:
Would this not prevent creating an ellipsoid around the zero point?

Reply:
hm, I suppose -- I'd taken that value since it was the default value for `center` from `clear`, but I suppose directly setting the started flag to `true` in the constructor would work as well

Reply:
Yea, I'd rather not use magic values that are in-bounds.